src/components/ProfileBio/ProfileBio.jsx

```diff
@@ -0,0 +1,19 @@
+import './ProfileBio.sass';
+
+import React, {Component} from 'react';
+
+export class ProfileBio extends Component {
+
+    render() {
+        const {} = this.props;
```

Comment:
Полагаю, `bio` и `name` тоже стоит вынести в props
```suggestion
        const {bio} = this.props;
```